apps/yarn.lock

```diff
@@ -60,7 +60,7 @@
     temporal latest
   optionalDependencies:
     browser-serialport latest
-    firmata "^0.12.0"
+    firmata latest
```

Comment:
Oh!  I'm referencing the `firmata` package from `CircuitPlaygroundBoard.js` when we don't depend on it directly in our package.json.  That's a mistake - it can cause use to resolve to different versions of that package in different places, meaning static state within that package isn't being shared when it should be.  I think if we depend directly on firmata and it satisfies the dependencies of both `johnny-five` and `playground-io` we should end up with only one copy which should fix the problem.  I'll set that all up then ping you back for review.